srtcore/packet.cpp

```diff
@@ -529,3 +533,71 @@ std::string PacketMessageFlagStr(uint32_t msgno_field)
 
     return out.str();
 }
+
+inline void SprintSpecialWord(std::ostream& os, int32_t val)
+{
```

Comment:
SPrint may be better, not to be confused with "sprint"

Reply:
The `Sprint` function exists already here. Maybe it's not the happiest name here, but I'd rather postpone the change of the name for this function globally (then both this one and the `Sprint` function would be renamed).

Reply:
ok